tsl/test/sql/dist_hypertable.sql.in

```diff
@@ -810,6 +852,8 @@ ORDER BY time;
 SELECT count(*) FROM twodim;
 $$);
 
+
+SET timescaledb.max_insert_batch_size=10;
```

Comment:
I don't see the added tests erroring out when COPY fails. We could maybe drop an attribute on a DN and then INSERT into that remote chunk and see COPY error out.

Reply:
Added.